Add WWN workaround for extra disks (#128)
harvester/docs: docs/host/host.md

1. Go to the **Hosts** page.
2. On the node you want to modify, click **⋮ > Edit Config**.
2. Select the **Disks** tab and click **Add Disks**.

> 3. Select an additional raw block device to add as an additional data volume.
>     - The `Force Formatted` option is required when the block device has once been force-formatted.

Reviewer: ```suggestion
3. Select an additional raw block device to add as an additional data volume.
    - The `Force Formatted` option is required when the block device has ever been force-formatted.
```

>     - The `Force Formatted` option is required when the block device has ever been force-formatted.

Reviewer 2: ```suggestion
    - The `Force Formatted` option is required if the block device has never been force-formatted.
```

!!! note

>     For Harvester to identify disks uniquely, it is required to have an unique WWN for each disk. Otherwise Harvester would refuse to add the disk.
>     If your disk does not have a WWN, you can also format it in EXT4 format to help Harvester identify it.

Reviewer: ```suggestion
    In order for the Harvester to uniquely identify the disks, each disk needs to have a unique WWN. Otherwise, Harvester will refuse to add the disk.
    If your disk does not have a WWN, you can also format it in `EXT4` format to help Harvester recognize it.
```

!!! note

>     If you are testing out Harvester within a QEMU environment, be aware of the version of QEMU itself. QEMU prior to v6.0 always generates the same WWN for NVMe disk emulations. Please use QEMU v6.0 or later to avoid such limitation.

Reviewer: ```suggestion
    If you are testing Harvester in a QEMU environment, please pay attention to the version of QEMU itself. QEMU prior to v6.0 always generated the same WWN for NVMe disk emulation. Please use QEMU v6.0 or later to avoid such limitations.
```

>     If you are testing Harvester in a QEMU environment, please pay attention to the version of QEMU itself. QEMU prior to v6.0 always generated the same WWN for NVMe disk emulation. Please use QEMU v6.0 or later to avoid such limitations.

Reviewer 2: ```suggestion
    If you are testing Harvester in a QEMU environment, you'll need to use QEMU v6.0 or later. Previous versions of QEMU will always generated the same WWN for NVMe disks emulation. This will cause Harvester to not add the additional disks, ask explained above.
```
